src/components/StaticApp/Header/Header.tsx

```diff
@@ -10,6 +10,29 @@ import { useThemeToggle } from 'components/ThemeProvider';
 
 import logo from 'assets/camino-logo-dark.png';
 import styles from 'components/Header/Header.module.scss';
+import useStyles from './styles';
+import { useAuthModalContext } from 'components/Auth/AuthModalProvider';
+
+const DemoBanner: FC = () => {
+  const styles = useStyles();
+  const { openLoginModal, openSignupModal } = useAuthModalContext();
+
+  return (
+    <div className={styles.root}>
+      <FontAwesomeIcon icon={faExclamationCircle} className={styles.icon} />
+      You are currently viewing syntheic demo data. Any changes made will not be saved. To create
+      your own pathways,{' '}
+      <span className={styles.underline} onClick={openLoginModal}>
+        login
+      </span>{' '}
+      or{' '}
```

Comment:
Does the space have to be in `{' '}`?  Maybe its doing something I don't understand.

Reply:
Everytime I deleted it the formatter just added it back. I removed it in a different editor and replaced them with spaces but the spaces don't render - so I guess it is required.
![Screen Shot 2020-11-12 at 2 12 38 PM](https://user-images.githubusercontent.com/53485517/98985134-2025d280-24f1-11eb-8cd5-8d4f32f227f0.png)